### PR TITLE
Use shared eslint

### PR DESCRIPTION
**Pull request by jbwyme**

### CODE REVIEW

**Comment by tdumitrescu** [comment]
What would this look like with only the linter updates?

File: lib/report/mp-app.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/*eslint camelcase: ["error", {properties: "never"}]*/
```
Comment (tdumitrescu) [inline lib/report/mp-app.js]: I don't think we should be throwing in a linter override for an entire code module like this (excepting cases like the constants module above). How many lines actually need this?

File: lib/util/index.js [hunk 2]
```diff
@@ -2,101 +2,102 @@ export * from './array';
...
 
   if (neg) {
-    commaized = '-' + commaized;
+    commaized = `-` + commaized;
   }
 
   return commaized;
 }
 
+
```
Comment (tdumitrescu) [inline lib/util/index.js]: extra blank line

File: package.json [hunk 3]
```diff
@@ -37,7 +37,8 @@
     "babel-polyfill": "6.9.1",
     "babel-preset-es2015": "6.9.0",
     "css-loader": "0.23.1",
-    "eslint": "2.11.1",
+    "eslint": "3.7.1",
+    "eslint-config-mixpanel": "^2.0.0",
```
Comment (tdumitrescu) [inline package.json]: bump to 2.0.1?

**Comment by tdumitrescu** [comment]
Not gonna do this, or reopening with fresh branch?

**Comment by jbwyme** [comment]
gonna do it but it was too out of date so gonna open fresh branch

**Pull request closed without merging**